Tribler/Core/Config/tribler_config.py

```diff
@@ -197,10 +197,10 @@ def get_permid_keypair_filename(self):
         return file_name
 
     def set_multichain_permid_keypair_filename(self, keypairfilename):
-        self.config['multichain']['ec_keypair_filename_multichain'] = keypairfilename
+        self.config['multichain']['ec_keypair_filename'] = keypairfilename
```

Comment:
This is why there is an `ec_keypair_filename` field under both the `multichain` and `general` sections.